File: Clases/Clase 3/Paquetes Java/Principales/Principal.java
package Principales;
import Matematicas.Operacion;

public class Principal{    
    public static void main(String[] args) {
        Operacion op1 = new Operacion();
        int a = 10;
        int b = 20;
        System.out.println("La suma de " + a + " + " + b + " es: " + op1.sumar(a, b));
    }
}

File: Clases/Clase 4/Paquetes C#/Principal.cs
using matematicas;
namespace principales{
    public class Principal{
        public static void Main(){
            System.Console.WriteLine("Metodo main");
            Operacion op = new Operacion();
            int a = 10;
            int b = 20;
            System.Console.WriteLine("La suma de " + a + " + " + b +  
                                    " es: " + op.sumar(a, b));
        }
    }
}